charts/sonarqube/README.md

```diff
@@ -147,13 +147,13 @@ The following table lists the configurable parameters of the Sonarqube chart and
 
 ### Image
 
-| Parameter | Description | Default |
-| --------- | ----------- | ------- |
-| `image.repository` | image repository | `sonarqube` |
-| `image.tag` | `sonarqube` image tag. | `9.2.4-{{ .Values.edition }}` |
-| `image.pullPolicy` | Image pull policy  | `IfNotPresent` |
-| `image.pullSecret` | (DEPRECATED) imagePullSecret to use for private repository | `None` |
-| `image.pullSecrets` | imagePullSecrets to use for private repository | `None` |
+| Parameter | Description | Default                       |
```

Comment:
Hey @alain-kermis-sonarsource, can you please remove all the extra spaces added here? While the markdown rendering is not impacted, the raw file it becomes less readable with all these spaces?